base/nvti.c

```diff
@@ -500,9 +505,37 @@ nvti_refs (const nvti_t *n, const gchar *type, const gchar *exclude_types,
 }
 
 /**
- * @brief Get the tag.
+ * @brief Get the solution.
  *
- * @param n The NVT Info structure of which the name should
+ * @param n The NVT Info structure of which the solution should
+ *          be returned.
+ *
+ * @return The solution string. Don't free this.
+ */
+gchar *
+nvti_solution (const nvti_t *n)
+{
+  return (n ? n->solution : NULL);
```

Comment:
Side note: you're the only one using parens with "return".  Everywhere else uses "return 0;" style.